UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -155,13 +153,14 @@ public static Mind NewSpawnPlayerV2(PlayerInfo account, Occupation requestedOccu
 	}
 
 
-
-	public static Mind NewSpawnCharacterV2(Occupation requestedOccupation, CharacterSheet character)
+	public static Mind NewSpawnCharacterV2(Occupation requestedOccupation, CharacterSheet character,
+		bool NonImportantMind = false)
```

Comment:
```suggestion
		bool nonImportantMind = false)
```

Reply:
also, what is a non important mind?

Reply:
Non Player Mind

Reply:
npcs will have mind?

Reply:
From what I've read, that seems to be Bod's intention. 
I don't really mind it because it makes behaviours consistent

Reply:
Yeah make that, it cleenes up when no mob body

Reply:
maybe ``nonPlayerMind`` then is better as a name

Reply:
though you might want some mobs to stick around , what do you think?